controllers/keyword.go

```diff
@@ -25,21 +24,6 @@ const (
 	uploadSuccessFlash = "CSV uploaded successfully"
 )
 
-var FilterList = []map[string]string{
```

Comment:
This list should not be moved to helpers. It's part of the controllers logic to sanitize user input. Helpers should be reserved for utilities. Its use should be limited. Now it seems to be the default "trash" can where you place everything 🙃 

Since it must be shared in both web and API controllers, then it's a sign that it might be part of the business logic. Services are where all the business logic so that's where this code should be placed in logically. 

In some cases, when user input and conditions vary, it can be also okay to create params or forms object store this outside of the services. For instance, in Beego and Rails, forms are used instead to process all user input data outside of services.

As a recommendation, avoid using helpers for your go-to place 💪

Reply:
Fixed in 218cb66832323f199af9783df05ad348aa9da7e3

Reply:
Still not fully satisfied that the filters list has been moved far from the controllers. But it works. A better solution would be to create other types of objects like forms or params.